packages/nimble-components/src/dialog/tests/dialog.spec.ts

```diff
@@ -242,7 +242,8 @@ describe('Dialog', () => {
         await disconnect();
     });
 
-    it('focuses the first button on the dialog when it opens', async () => {
+    // see https://github.com/ni/nimble/issues/1075
```

Comment:
Can you be more explicit in the link

```suggestion
    // Firefox skipped, see: https://github.com/ni/nimble/issues/1075
```

Reply:
Done